protocol/x/clob/keeper/liquidations.go

```diff
@@ -360,11 +360,22 @@ func (k Keeper) IsLiquidatable(
 		return false, err
 	}
 
-	// The subaccount is liquidatable if both of the following are true:
-	// - The maintenance margin requirements are greater than zero (note that they can never be negative).
-	// - The maintenance margin requirements are greater than the subaccount's net collateral.
-	isLiquidatable := bigMaintenanceMargin.Sign() > 0 && bigMaintenanceMargin.Cmp(bigNetCollateral) == 1
-	return isLiquidatable, nil
+	return CanLiquidate(bigNetCollateral, bigMaintenanceMargin), nil
+}
+
+// CanLiquidate returns true if the subaccount is able to be liquidated given the total net collateral
+// and maintenance margin requirement of the subaccount.
+//
+// The subaccount is liquidatable if both of the following are true:
```

Comment:
```suggestion
// The given TNC, MMR pair is liquidatable if both of the following are true:
```

Reply:
hmmm I think calling TNC/MMR pair liquidatable is slightly weird. 

Going to leave this as is if that's okay but I renamed the function to `CanLiquidatateSubaccount`, hopefully this docstring is more appropriate now